apis/fluentbit/v1alpha2/plugins/filter/lua_types.go

```diff
@@ -76,12 +79,17 @@ func (l *Lua) Params(_ plugins.SecretLoader) (*params.KVs, error) {
 		kvs.Insert("type_int_key", strings.Join(l.TypeIntKey, " "))
 	}
 
+	if l.TypeArrayKey != nil && len(l.TypeArrayKey) > 0 {
+		kvs.Insert("type_array_key", strings.Join(l.TypeIntKey, " "))
```

Comment:
should be `kvs.Insert("type_array_key", strings.Join(l.TypeArrayKey, " "))` ?

Reply:
ah, right, my mistake. fixing.

Reply:
@benjaminhuo please check again, now that should be fixed.